### PR TITLE
Caps And Crafting

### PR DESCRIPTION
**Pull request by chowdermcarthor**
Simplifies the trading outpost, both in use and in code; makes it far easier to both sell items, and add values to items in order to be sold, in order to better stimulate a healthy caps economy.
Also adds back in wasteland food crafting.

### CODE REVIEW

**Comment by chowdermcarthor** [comment]
The heck does this even *mean*

**Comment by Superballs69** [comment]
Howdy! Please resolve the CI checks involving finding space indentations

` code/datums/components/crafting/recipes/wastelandfood.dm:    	/obj/item/reagent_containers/food/snacks/meat/steak/squirrel = 1,
code/datums/components/crafting/recipes/wastelandfood.dm:    	/obj/item/reagent_containers/food/snacks/grown/carrot = 1,
code/datums/components/crafting/recipes/wastelandfood.dm:    	/obj/item/reagent_containers/food/snacks/grown/potato = 1,
code/datums/components/crafting/recipes/wastelandfood.dm:    	/obj/item/reagent_containers/glass/bowl = 1`

Then fill the changes out using the template that was originally given to you when you made the PR.

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -157,6 +157,7 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 
 	var/canMouseDown = FALSE
 
+	var/Value = 0 //For purposes of selling things at the wasteland vendor
```
Comment (MarinaGryphon) [inline code/game/objects/items.dm:160]: don't capitalise vars. use `var/caps_value` or something.

**Review by MarinaGryphon — CHANGES_REQUESTED** [review]

**Comment by chowdermcarthor** [comment]
> Howdy! Please resolve the CI checks involving finding space indentations
> 
> ` code/datums/components/crafting/recipes/wastelandfood.dm: /obj/item/reagent_containers/food/snacks/meat/steak/squirrel = 1, code/datums/components/crafting/recipes/wastelandfood.dm: /obj/item/reagent_containers/food/snacks/grown/carrot = 1, code/datums/components/crafting/recipes/wastelandfood.dm: /obj/item/reagent_containers/food/snacks/grown/potato = 1, code/datums/components/crafting/recipes/wastelandfood.dm: /obj/item/reagent_containers/glass/bowl = 1`
> 
> Then fill the changes out using the template that was originally given to you when you made the PR.

Alright, I'll work on this today - I haven't had time.
Do you want me to list every crafting item added? It's just a port of the old ones from before the rebase.

**Pull request closed without merging**